### PR TITLE
return a coreschema.Number if the field filter is a NumberFilter

### PR DESCRIPTION
**Pull request by willemarcel**
in the `get_schema_field()`, return a `coreschema.Number` if the field filter is a `NumberFilter`

### CODE REVIEW

**Comment by rpkilby** [comment]
@carltongibson - should there be handling for additional types? eg, dates, csv list of numbers., etc..

**Comment by carltongibson** [comment]
@rpkilby Yes. But I'm happy to evolve to it.

**Comment by rpkilby** [comment]
Given that it's a behavior change (string => number), should this be milestoned for 1.1?

File: django_filters/rest_framework/backends.py [hunk 1]
```diff
@@ -67,6 +67,13 @@ def to_html(self, request, queryset, view):
 
         return template.render(context, request)
 
+    def get_coreschema_field(self, field):
+        if isinstance(field, filters.NumberFilter):
+            field_cls = compat.coreschema.Number
+        else:
+            field_cls = compat.coreschema.String
+        return field_cls(description=six.text_type(field.extra.get('help_text', ''))
```
Comment (rpkilby) [inline django_filters/rest_framework/backends.py]: You forgot this `)` 😉

Comment (carltongibson) [inline django_filters/rest_framework/backends.py]: On it. :-) 

Silly GitHub editing UI.

**Comment by carltongibson** [comment]
Closing in favour of #700. @willemarcel Thanks for the input!

**Pull request closed without merging**